### ISSUE TITLE
Data Augmentation (regression)

### ISSUE THREAD

**Issue by eric-bueno** [broadinstitute/segmentation_experiments#37]
* Downscale images that are equal to a given bin to every bin below it
    * only upscale to no more than 10 bins above it
* Generate more crops for larger bins with less examples
* Plot distribution of sizes after all these transformations
    * ideally, sizes should be uniformly distributed

**Comment by eric-bueno** [comment]
* Downscaling images to every bin below it didn't work out because downscaling beyond a certain point made it impossible to get high quality cropped images.
* Rescaled dataset is much more uniform than initial dataset
  * We find which images are compatible to each bin in our distribution and rescale each of these images to match the corresponding bin
* Added comments to codebase
# Reports on original and augmented sets
* Original (30 epochs)
-- mean_absolute_error ~26 units
-- validation mean_absolute_error ~ 256 units
<img width="1346" alt="Screen Shot 2020-09-28 at 11 51 03 AM" src="https://user-images.githubusercontent.com/66489438/94455318-ea1ad200-0180-11eb-9370-49177c848f51.png">

* Augmented (80 epochs)
-- mean_absolute_error ~64 units
-- validation mean_absolute_error ~365 units
<img width="1343" alt="Screen Shot 2020-09-28 at 1 31 23 PM" src="https://user-images.githubusercontent.com/66489438/94466127-ef7f1900-018e-11eb-9522-3d38093a7a28.png">

Although the augmented data set has a more even distribution of nuclei sizes, it has a much lower accuracy even with nearly 3x the training cost. Clearly, the neural network must be modified to take advantage of the homogeneity of the dataset.

Progress has been made in commit 7acf7b2b270a820ffbb5e391df2988e3815426f5
